### PR TITLE
Update '--wrapper' man

### PR DESCRIPTION
**Pull request by ethanalker**
Fixes the '--wrapper' comment in the manpage suggesting to use with 'i3 exec', since it has been replaced with '--i3-ipc'.

### CODE REVIEW

File: j4-dmenu-desktop.1 [hunk 1]
```diff
@@ -76,7 +76,8 @@ Performing
 will exit the program.
 .It Fl Fl wrapper Ar wrapper
 A wrapper binary.
-Useful in case you want to wrap into 'i3 exec'.
+Using to wrap into 'i3 exec' is deprecated. See
+.Fl Fl i3-ipc .
```
Comment (meator) [inline j4-dmenu-desktop.1]: I am not a native English speaker, but this doesn't look like valid English to me. Please also validate manpages with `man -T lint ./j4-dmenu-desktop.1` (this works with `mdocml`, I don't know if it works for `man-db`). Sentences need to be separated by a newline.

The change could look something like this (inspired by `--skip-i3-exec-check`'s description):

```suggestion
Usage of
.Ql --wrapper Qq i3 exec
and
.Ql --wrapper Qq sway exec
is deprecated, use
.Fl Fl i3-ipc
instead.
```

**Review by meator — COMMENTED** [review]
Thanks for fixing the `--wrapper` description, I forgot to fix it after implementing i3 IPC.

[`main.cc:169`](https://github.com/enkore/j4-dmenu-desktop/blob/r3.1/src/main.cc#L169) should also be updated.

**Comment by ethanalker** [comment]
Added the changes to [main.cc:169](https://github.com/enkore/j4-dmenu-desktop/blob/r3.1/src/main.cc#L169). The completion scripts look like they can stay as they are.

**Comment by ethanalker** [comment]
I wasn't sure about formatting for '--help', so I just tried to match the other help messages.

**Pull request merged**